### PR TITLE
[#24] Connect landing page to GET moments endpoint

### PR DESCRIPTION
**Pull request by tehblasian**

### CODE REVIEW

File: frontend/src/pages/LandingPage.js [hunk 1]
```diff
@@ -1,19 +1,60 @@
 import React, { Component } from 'react';
+import axios from 'axios';
 import Moment from '../components/Moment';
-import sample from '../resources/sample-moment.jpg';
-import logo from '../resources/logo.svg';
-import tai from '../resources/TaiLopez.png';
+// import sample from '../resources/sample-moment.jpg';
+// import logo from '../resources/logo.svg';
+// import tai from '../resources/TaiLopez.png';
```
Comment (ZacharyBys) [inline frontend/src/pages/LandingPage.js]: These can probably be removed

File: frontend/src/pages/LandingPage.js [hunk 2]
```diff
@@ -1,19 +1,60 @@
...
+    componentWillMount(){
+        axios.get('http://localhost:16085/api/moments').then(response => {
+            let moments = response.data.moments;
+            this.setState({
+                moments,
+            });
+        }).catch(error => {
+            console.log(error);
+            this.setState({
+                error: 'Oops! Something went wrong...'
```
Comment (ZacharyBys) [inline frontend/src/pages/LandingPage.js:23]: Maybe a more descriptive error, your call though

**Review by ZacharyBys — DISMISSED** [review]
Moments ordered by date that's good.
You can choose to use my suggestions, but they aren't crucial.

**Review by ZacharyBys — APPROVED** [review]

File: frontend/src/pages/LandingPage.js (hunk 2 above)
Comment (tehblasian) [reply frontend/src/pages/LandingPage.js:23]: I'd argue that the user doesn't necessarily need to know the details of the error.

**Pull request merged**